app/views/shared/_site_navbar.html.erb

```diff
@@ -4,15 +4,15 @@
       <ul class="site-search-nav navbar-nav mr-auto mr-md-3 mr-lg-auto w-100 order-1 order-md-0 mt-3 mt-md-0">
         <li class="navbar-item d-flex flex-grow-1">
           <% if Settings.feature_flags.search_across || in_search_across? %>
-            <%= render 'shared/site_search_form', presenter: Blacklight::SearchBarPresenter.new(controller, SearchAcrossController.blacklight_config) %>
+            <%= render SiteSearchFormComponent.new %>
           <% end  %>
         </li>
       </ul>
     <% else %>
       <ul class="navbar-nav mr-auto w-100 order-1 order-md-0 mt-3 mt-md-0">
         <li class="navbar-item">
           <% if Settings.feature_flags.search_across || in_search_across? %>
-            <%= render 'shared/old_site_search_form', presenter: Blacklight::SearchBarPresenter.new(controller, SearchAcrossController.blacklight_config) %>
+            <%= render OldSiteSearchFormComponent.new %>
```

Comment:
Can we delete this whole clause? The feature flag is true in prod.  I think we can remove it now and get rid of all of this code.https://github.com/sul-dlss/shared_configs/blob/exhibits-prod/config/settings/production.yml#L30C3-L30C17

Reply:
Double checked. It looks like we can. PR has removed this.